styles.css

```diff
@@ -31,17 +31,46 @@ body {
     background-color: #e5e5e5;
   }
 
-  .menu {
+  .sections {
     display: none;
+    top: 0;
+    left: 100%;
+    transition: left 0.8s, visibility 0.8s, opacity 0.4s;
+    flex-direction: column;
+    align-items: flex-start;
   }
 
-  #intro {
+  .intro {
+    width: 100%;
+  }
+
+  .hamb {
+    position: fixed;
+  }
+
+  .active {
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+    background-color: #e5e5e5;
+    filter: blur(1px);
+    width: 100%;
+    left: 0;
+    height: 610px;
   }
 
-  .intro {
+  .active a {
+    color: #000;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 35px;
+    filter: blur(1px);
     width: 100%;
+    left: 0;
+    height: 610px;
   }
 
```

Comment:
- [ ] for your menu items, Please make sure the list items are the same as the ones on the figma design. this includes `font-family`, `font-size`, and `text-decoration`/remove the underline, and please no blue effect on the text.
- [ ] Also, when the menu opens, I notice the close/`X` icon is not fully visible. Please address this to have a better layout and provide a better user experience. 

### Your design 
 
![image](https://user-images.githubusercontent.com/69077061/223529077-15adb09a-8510-4577-aeed-879fd4508a7e.png)

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/223529223-03b4d05c-fd1c-4b95-821c-ce6be91a4aa6.png)